.build/package.json

```diff
@@ -25,6 +25,8 @@
     "grunt-contrib-cssmin": "^0.14.0",
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-contrib-sass": "^0.9.2",
+    "grunt-contrib-watch": "^0.6.1",
+    "grunt-contrib-qunit": "^0.7.0",
```

Comment:
I'm pretty sure we don't use qunit and neither does Sickrage/Sickrage so why was this added?